eng/pipelines/mono/templates/build-job.yml

```diff
@@ -86,6 +102,39 @@ jobs:
         artifactName: $(buildProductArtifactName)
         displayName: 'product build'
 
+    # Build packages
+    - ${{ if and(ne(parameters.llvm, true), ne(parameters.osGroup, 'Windows_NT')) }}:
```

Comment:
Any reason to explicitly disable the LLVM packages?

Reply:
Just to keep problem scope in check. Can always follow up with LLVM packaging.